docs/development-and-testing.md

```diff
@@ -1,34 +1,56 @@
 ## Development and Testing
-0. You would need to have `quay.io` account, running docker and kubernetes; and make sure `quay.io/<QUAY_USERNAME>/direct-csi` repository is `public`.
-1. Go to project root
+1. You would need to have `quay.io` account, running docker and kubernetes.
```

Comment:
This is not a requirement. registry can be configured using `--registry`

Reply:
Fixed in different way.